plugins/admission/pkg/admission/admission.go

```diff
@@ -57,7 +58,7 @@ func SendResults(reports []models.ReportInfo, token string) (passed bool, warnin
 
 	req.Header.Set("Content-Type", w.FormDataContentType())
 	req.Header.Set("Authorization", "Bearer "+token)
-
+	req.Header.Set("X-Fairwinds-Admission-Version", admissionversion.String())
 	for _, report := range reports {
 		req.Header.Set("X-Fairwinds-Report-Version-"+report.Report, report.Version)
```

Comment:
I think this `report.Version` gets set somewhere in the admission package - are we able to update OPA here?

Reply:
Yes - I've just pushed an update that adds `embed.go` to the OPA plugin and uses it within the Admission plugin / opa package for the OPA report.

Once we verify we like how this looks I think we can do the same for other reports that Admission uses.